webrtc.html

```diff
@@ -4218,6 +4218,8 @@
               </tbody>
             </table>
           </div>
+          <p>Note: the user agent will typically only gather <code>active</code> 
+          ICE TCP candidates.</p>
```

Comment:
May want to include some reasoning, like: "because `passive` candidates typically only work if both peers are on the same LAN, and `so` candidates may not be supported by the operating system." I assume that's the reason. Or have we just not got around to implementing it? Maybe @juberti or @pthatcherg know.